logger/logger.go

```diff
@@ -71,3 +44,84 @@ func GetLevel(levelStr string) (Level, error) {
 	}
 	return InfoLevel, fmt.Errorf("Unknown Level String: '%s', defaulting to NoLevel", levelStr)
 }
+
+// basic Logger is the default global
+var globalLogger Logger = NewLogger()
+
+func SetGlobalLogger(logger Logger) {
+	globalLogger = logger
+}
+
+func SetGlobalLevel(lvl Level) {
+	globalLogger.SetLevel(lvl)
+}
+
+func Info(args ...interface{}) {
+	globalLogger.Log(InfoLevel, "", args, nil)
+}
+func Infof(template string, args ...interface{}) {
+	globalLogger.Log(InfoLevel, template, args, nil)
+}
+func Infow(msg string, fields Fields) {
```

Comment:
Do we need the "*w" methods?

Reply:
well, the `Fields` has to be added per log event.
if you set `Fields` in logger , Then it is  considered as `sub logger` which is reused for all subsequent logging events. that may not what developer want. 
there is also need for creating sub loggers (example developer want add `category=database` field for all logs form `data access layer`)